docs/recipes/sru.rst

```diff
@@ -0,0 +1,118 @@
+Loading Data via SRU
+====================
+
+Polymatheia supports accessing metadata records via the `SRU`_ protocol. SRU (Search/Retrieve via URL) is
+a standard XML-based protocol for search queries, utilizing CQL (Contextual Query Language),
+a standard syntax for representing queries. Each web service that implements the SRU protocol should
+provide an `Explain record`_ at its base URL that allows a client to retrieve a
+description of the facilities available at this SRU server.
+
+.. _`SRU`: http://www.loc.gov/standards/sru/
+.. _`Explain record`: http://www.loc.gov/standards/sru/explain/
+
+Getting the Explain record
+--------------------------
+
+Use the :class:`~polymatheia.data.reader.SRUExplainRecordReader`. Polymatheia provides direct
+access to the record schemas that can be used with the SRU web service as well as to the echoed request
+(i.e., the request parameters echoed back to the client).
+
+.. sourcecode:: python
+
+    from polymatheia.data.reader import SRUExplainRecordReader
+
+    reader = SRUExplainRecordReader("http://sru.k10plus.de/gvk")
+    for record in reader:
+        print(record)
+
+    print(reader.schemas)
+    print(reader.echo)
+
+Fetching records
+----------------
+
+Use the class :class:`~polymatheia.data.reader.SRURecordReader` to query an SRU server.
+Provide a parameter ``max_records`` that specifies the desired number of records to return:
+
+.. sourcecode:: python
+
+    from polymatheia.data.reader import SRURecordReader
+
+    reader = SRURecordReader("http://sru.k10plus.de/gvk",
+                             query="dog cat mouse",
+                             max_records=10)
```

Comment:
What happens if you don't specify max_records. Does it return all records? If yes, then I would suggest splitting the recipe into two. One to fetch all records (with the minimum amount of code) and one to fetch a limited number of records.